README.md

```diff
@@ -221,6 +221,20 @@ products:
     dropdown: true
 ```
 
+5. Normalized Words List
+
+Each application could have their own list of custom words that are meant to be all UPPERCASE or all DOWNCASE, instead of title casing. Those words are stored inside `/config/normalized_words.yml`:
```

Comment:
`DOWNCASE` being in uppercase confused me for a second here

Reply:
Oops, that is confusing! I'll amend.